### DISCUSSION TITLE
ddospot service in T-POT is in "DOWN" status and all the servies are going down.

### DISCUSSION THREAD

**Discussion by muhtyo1102** [telekom-security/tpotce#1453] [General]
Dear, experts.

I configured T-pot on my AWS EC2 instance (t2.large),

But executing the dsp.sh, only the ddospot service is not going up and after a while all the services are down. 
(I attached the images.)

Checking the /var/log/syslog, the follwing message appears.

[Dec 26 08:00:30 superpajamas docker-compose[42626]: ERROR: for ddospot  Cannot start service ddospot: driver failed programming external connectivity on endpoint ddospot (2a80d15f045fdb5aa3b0011f94a3171a383c87b21f73acdef47d0c6ca36d1641): Error starting userland proxy: listen udp4 0.0.0.0:53: bind: address already in use]

Does someone have any good idea to solve this issue ?

<img width="1109" alt="image" src="https://github.com/telekom-security/tpotce/assets/65376011/8167f128-413a-4f8b-a326-9ee48d3e8061">

<img width="467" alt="image" src="https://github.com/telekom-security/tpotce/assets/65376011/6556b445-551a-49a3-9c5e-e945771f813b">

**Comment by evoke0** [comment]
As it says, port 53 is already in use by any other service. You can check what ports are being in use before starting tpot with `systemctl stop tpot` and run `netstat -tulpen`. Attach results

**Reply by muhtyo1102** [reply]
Dear, evoke0

Thank you so much. I could resoleved this issue.

**Reply by asolares-ahold** [reply]
how did you solve this issue?

**Reply by muhtyo1102** [reply]
To stop local DNS stub listner on Ubuntu.

I referred this article. But this is a Japanese article. I hope you can translate with some technique.
https://qiita.com/shora_kujira16/items/31d09b373809a5a44ae5
